### ISSUE TITLE
Move registration agreement/terms of service into registration form

### ISSUE THREAD

**Issue by rjtngit** [wetfish/online#66]
Doesn't need an entire page to itself. Too many clicks!

**Comment by itsrachelfish** [comment]
I agree, it's very scary. I think it should be like "Checking this box means you agree to the terms of service." and if you click on the terms of service is a link it expands to show the agreement text.
